api/lib/domain/models/UserLogin.js

```diff
@@ -25,35 +25,35 @@ class UserLogin {
     this.temporaryBlockedUntil = null;
   }
 
-  shouldBlockUserTemporarily() {
+  shouldMarkUserAsTemporarilyBlocked() {
     return this.failureCount % settings.login.temporaryBlockingThresholdFailureCount === 0;
   }
 
-  blockUserTemporarily() {
+  markUserAsTemporarilyBlocked() {
     const commonRatio = Math.pow(2, this.failureCount / settings.login.temporaryBlockingThresholdFailureCount - 1);
     this.temporaryBlockedUntil = new Date(Date.now() + settings.login.temporaryBlockingBaseTimeMs * commonRatio);
   }
 
-  hasBeenTemporaryBlocked() {
+  hasBeenMarkedAsTemporaryBlocked() {
     return this.failureCount > 0 || !!this.temporaryBlockedUntil;
   }
 
-  shouldBlockUser() {
+  shouldMarkUserAsBlocked() {
     return this.failureCount >= settings.login.blockingLimitFailureCount;
   }
 
-  blockUser() {
+  markUserAsBlocked() {
     this.blockedAt = new Date();
   }
 
-  unblockUser() {
+  resetUserBlocking() {
     this.failureCount = 0;
     this.temporaryBlockedUntil = null;
     this.blockedAt = null;
   }
 
-  isUserBlocked() {
-    return !!this.blockedAt || this.failureCount >= settings.login.blockingLimitFailureCount;
+  isUserMarkAsBlocked() {
```

Comment:
```suggestion
  isUserMarkedAsBlocked() {
```

On pourrait même le transformer en getter.